subjects/sumascii/README.md

```diff
@@ -0,0 +1,18 @@
+## sumascii
+
+### Instructions
+
+Create a program that receives an argument and displays the sum of the ASCII value of each character. The sum will be a number representing the value after the sum of the bytes. If there is an invalid input print 0.
+
+### Usage
+
+```console
+$ go run . "hi" | cat -e
+209$
+$ go run . "" | cat -e
+0$
+$ go run . "S" "A"
```

Comment:
Regarding the first comment, it's missing a return. I'll fix that, thank you.
Regarding the second, In my opinion it is correct since we are adding bytes, as I explained on the comment more to the bottom, please let me know what you think.